docs/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- Implement collector for ps_constant project (OSIDB-1199)
```

Comment:
nitpick: the project is I think called `ps-constants` (https://git.prodsec.redhat.com/devops/ps-constants/)